autometa/common/external/samtools.py

```diff
@@ -37,32 +56,41 @@ def run(cmd):
     logger.debug(f'cmd: {cmd}')
     with open(os.devnull, 'w') as stdout, open(os.devnull, 'w') as stderr:
         retcode = subprocess.call(cmd, stdout=stdout, stderr=stderr, shell=True)
-    if retcode:
+    if not retcode:
+        return True
+    else:
         logger.warning(f'args:{cmd} retcode:{retcode}')
         return False
-    return True
```

Comment:
I think it is clearer using `if retcode` versus `if not retcode`

Similarly an else statement is, in this case, redundant because if the "if" statement is executed, the function will return a value. Jason's comments on the return values here are specific to programming conventions. Since the return values are not used, you could return a NoneType object and in the case of the `run` command failling, raise a ChildProcessError instead of a logger warning.

You could tackle this two ways:

1. If there is some issue, I.e. a return code that is not equal to 0, then raise a ChildProcessError (this is a python built-in) exception. Note: The docstring will need to be updated to reflect this Exception.
2. Log the exception via logger.exception('same string as the logger.warning string') and allow Autometa to continue the analysis. 

**My preference lies with number 1.**